src/main/resources/reference.conf

```diff
@@ -869,6 +869,29 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  catalog = {
+    actionPatterns = {
+      "create_metadata" = {
+        description = "Can create catalog metadata"
+      },
+      "read_metadata" = {
+        description = "Can read catalog metadata"
+      },
+      "update_metadata" = {
+        description = "Can update catalog metadata"
+      },
+      "delete_metadata" = {
```

Comment:
I'm wondering if we need different permissions for create/update/delete separately. I can't think of a reason now why we would, but I guess there's no great harm in adding them all.

Reply:
I thought it would be clearer this way, but since that admin role is at the catalog level it may not matter

Reply:
In general the pattern for Sam is to make permissions as granular as possible, so from a pattern perspective this is correct for the repo as well.